com.unity.render-pipelines.core/Runtime/Unity.RenderPipelines.Core.Runtime.asmdef

```diff
@@ -30,6 +30,11 @@
             "name": "com.unity.rendering.hybrid",
             "expression": "0.6.0-preview.0",
             "define": "HYBRID_0_6_0_OR_NEWER"
+        },
+        {
+            "name": "com.unity.rendering.hybrid",
+            "expression": "0.14.0-preview.0",
+            "define": "ENABLE_HYBRID_RENDERER_V2"
```

Comment:
I don't think this is actually needed in the end. No harm in keeping it for now though

Reply:
Yes, it seems it shouldn't be needed, but since one file is using it in the assembly I just added it here as well.
The file in question for the record : https://github.com/Unity-Technologies/Graphics/blob/10.x.x/hrv2-default/com.unity.render-pipelines.core/Runtime/AssemblyInfo.cs